ckan/templates/home/index.html

```diff
@@ -8,7 +8,7 @@
 
 {% block content %}
   <div class="homepage layout-{{ homepage_style }}">
-    <div class="container">
+    <div id="content" class="container">
```

Comment:
This is good for the home page.  It seems for most other pages this is already set in [page.html](https://github.com/ckan/ckan/blob/0e3f501e9fe36289cee4fc59647db80feb0e2f97/ckan/templates/page.html#L21).

One thing that seems to be missing is making the skip link keyboard accessible.  Right now it seems that you can't navigate to it and click on it. It looks like the current way it's being [hidden is an old way of just using `display:none;.`](https://github.com/ckan/ckan/blob/18adddcf1150f912b65bac5aa439e5bd647a52ee/ckan/templates/page.html#L6).

Would be good to switch this to Bootstraps built in handling of this as [outline here.](https://getbootstrap.com/docs/3.3/getting-started/#skip-navigation). 

As is, it'll pass an automated test but not a manual one.

Reply:
@Zharktas including you for your input and not sure if you'd get notified on a close PR?

Reply:
@boykoc yeah, it should use sr classes instead of hide. Want to make pr for it ?